sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/implementation/http/policy/DefaultRedirectStrategy.java

```diff
@@ -60,7 +60,7 @@ private DefaultRedirectStrategy(int maxAttempts, HeaderName locationHeader, Set<
 
     private static HeaderName validateLocationHeader(String locationHeader) {
         if (CoreUtils.isNullOrEmpty(locationHeader)) {
-            LOGGER.log(ClientLogger.LogLevel.ERROR, () ->
+            LOGGER.atError().log(() ->
                 String.format("'locationHeader' provided as null will be defaulted to {%s}", HeaderName.LOCATION));
```

Comment:
This doesn't need to be a string format as the supplied string is a constant, should initialize this once and use that every time.